gears/src/signing/std_sign_doc.rs

```diff
@@ -0,0 +1,114 @@
+use serde::{Deserialize, Serialize};
+use serde_json::{Map, Value};
+
+// gears::core::base::coin::Coin has wrong order of fields
+// It is better to create a struct with correct order than
```

Comment:
I don't think there would be a problem with changing the order in gears Coins [here](https://github.com/rumos-io/gears/blob/72baade2fd17cd9a91c7fc83f74a662544352f0a/gears/src/types/base/coin/unsigned.rs#L18)

Reply:
The structure requires strings instead of domain types. May I add custom serialization/deserialization for struct like`#[serde(try_from`?

Reply:
Why does the structure require Strings? Doesn't it just need to serialize to JSON?